src/engine/bank01.asm

```diff
@@ -8254,7 +8292,11 @@ PlayMoveAnimation: ; 7494 (1:7494)
 	INCROM $74dc, $7571
 
 Func_7571: ; 7571 (1:7571)
-	INCROM $7571, $7576
+	rst $28
```

Comment:
`rst $28` is a very special case in TCG. The handler for `rst $28` does some manipulation of the stack, so that the next three bytes after the `rst $28` instruction are not asm instructions like they usually would be.
Take a look at the `farcall` macro in `src/macros/code.asm` to see what I mean.
Instead of using `gbz80disasm.py`, you should use `tcgdisasm.py` (in this repo in the `tools/` directory) and it will handle this special case (and other special cases) automatically.

Reply:
You can also look at the `rst $28` handler in `src/engine/home.asm` to see what I mean by stack manipulation.